swim/member_test.go

```diff
@@ -74,52 +75,6 @@ func newChange(addr string, s state) Change {
 	}
 }
 
-func (s *MemberTestSuite) TestNonLocalOverride() {
```

Comment:
YES! I totally thought that I had a test for that function, but it is such a crucial part of the protocol all paths should be tested for `acceptGossip`. Thanks!